website/public/js/controllers/groupsCtrl.js

```diff
@@ -47,6 +47,11 @@ habitrpg.controller("GroupsCtrl", ['$scope', '$rootScope', 'Shared', 'Groups', '
       group._editing = false;
     };
 
+    $scope.cancelSave = function(group) {
+      group = group.$get();
```

Comment:
Does the `$get` method return the group object? I thought `group.$get` returned a promise.